drivers/storage/portworx/component/csi.go

```diff
@@ -460,6 +460,24 @@ func getCSIDeploymentSpec(
 	}
 	imagePullPolicy := pxutil.ImagePullPolicy(cluster)
 
+	var args []string
+	if util.GetImageMajorVersion(provisionerImage) >= 2 {
```

Comment:
Would it be possible to add a test for this case where the provisioner image version is greater than 2.x?

Reply:
Yeah I'll add one - I think maybe via the desiredimages field.

Reply:
I don't think this is possible for now via the desired images, seems to just get reset. 

Once we need to start support 2.x csi provisioner, this will have coverage.